exetera/core/fields.py

```diff
@@ -112,6 +112,32 @@ def _ensure_valid(self):
         if not self._valid_reference:
             raise ValueError("This field no longer refers to a valid underlying field object")
 
+    def where(self, cond, a, b):
```

Comment:
Numpy-like exetera.where would need to live in the top-level module namespace as a function
Pandas-like Field.where and DataFrame.where should be on the abstract types
As Eric points out, pandas supports the callable but we should also support Field and ndarray/array-like as inputs

Reply:
Do we just want to drop the numpy-like exetera.where?